packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -314,13 +339,9 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
           </ComboBoxInputContainer>
           <ComboBoxDropdown {...dropdownMenuProps}>
             <ComboBoxMenu>
-              {filteredOptions.length > 0 ? (
-                filteredOptions.map(getMenuItem)
-              ) : (
-                <MenuExtraContent>
-                  <Text isMuted>{emptyStateMessage}</Text>
-                </MenuExtraContent>
-              )}
+              {filteredOptions.length > 0
```

Comment:
Maybe need to check for virtualization here as it doesn't need all those extra menu items in `children`

```suggestion
              {filteredOptions.length > 0 && !enableVirtualization
```

Reply:
Added